### DISCUSSION TITLE
Windows美化历程

### DISCUSSION THREAD

**Discussion by GuZhengSVT** [share121/inter-knot#4602] [教程]
## 前言的前言
第一次来绳网，搬运一下我博客最近写的一篇文章作为测试。
祝理性、客观的中文社区越来越好。

## 前言
俗话说的好，**美化的尽头是默认**。
虽然默认的 Windows 已经能高效地胜任各项工作，但它确实 ugly 啊。
在拥有一台性能过剩 PC 的情况下，适度追求美化和简化，满足本私斋的~~高端审美~~是极有必要的（）

下面来说说我在用/用过的美化软件/方案。

## 当前桌面方案
- TranslucentTB：任务栏透明/亚克力效果
- Sapphire：桌面图标交互优化
- Wallpaper Engine：为减少 GPU 消耗和内存占用，只选了 Blue Archive 中 Noa 的记忆大厅作为壁纸，几秒的 4 K 视频，整个 Wallpaper 内存占用约 100 Mb。
- Rainmeter：只用了一个音频条作为美化，因为 Noa 已经够美了😋
  原来用过查看硬件信息的组件，发现除了增加焦虑外没什么用，遂放弃。
- QQ 美化、网易云音乐美化、光标美化、Obsidian 主题、插件安排上。再加上 Edge 浏览器的 iTab 标签页和各类实用插件，目前的工作流完全舒适。

贴几张图
![桌面展示1](https://github.com/user-attachments/assets/cdb15cef-2680-4ff0-a683-37a87bb81d7e)

![桌面展示2](https://github.com/user-attachments/assets/5f1f8d18-3420-449d-8a8d-f6bd8175f88a)

![Edge展示](https://github.com/user-attachments/assets/2eafd82a-8dfc-427f-b3ec-152e4ba1b351)

![Obsidian展示](https://github.com/user-attachments/assets/1055cacc-29ed-4b15-90cb-aa8d8afa4d3b)

![任务管理器展示](https://github.com/user-attachments/assets/240bf4a1-5926-4a49-8323-74506d6a2561)

在以上正常工作流下，内存占大头的是浏览器，GPU 消耗主要来自 wallpaper，从登录到完全自启动耗时 10s 内，均在可接受负载范围内。

**配置：**
- 12400 F
- 7700 XT
- 32G DDR4
- 2K 180Hz HDR 显示器
- Windows 11 专业版 23H2

## QQ 美化
==特别推荐！==
QQ 是大部分中国人不得不用又恨之入骨的东西——广告、弹窗、不想看的娱乐页面、花里胡哨功能性差的界面。
Windows 端自从推出 QQ 9，上述情况改善了不少，然而恶心人的东西最近逐渐加回来了，只能说本性难移。
为了不被迫在这个常用软件上天天吃屎，伟大的具有开源精神和折腾精神的中文互联网人纷纷投入 QQ 改造计划，而今天我要介绍的就是其中一个伟大项目——**LiteLoaderQQNT**。

效果图：
![QQ美化展示](https://github.com/user-attachments/assets/29f436cd-dd33-49be-a9ca-a6efb72afc94)

好了说正事，liteloader 是 QQNT 的插件平台，安装后可以下载众多插件。
GitHub 项目地址：[LiteLoaderQQNT: QQNT 插件加载器：轻量 · 简洁 · 开源 · 福瑞](https://github.com/LiteLoaderQQNT/LiteLoaderQQNT)

部分推荐插件如下：
1. [PluginInstaller](https://github.com/xinyihl/LiteLoaderQQNT-PluginInstaller/tree/main)：LiteLoaderQQNT 插件安装器，可检查更新和一键安装/重启。先安装这个再安装其他插件省事很多。
2. [LL-plugin-list-viewer](https://github.com/ltxhhz/LL-plugin-list-viewer/tree/main): LiteLoaderQQNT Plugin 插件列表查看·安装·更新。收录了大部分插件，可直达 GitHub 项目地址。安装功能存在问题，部分插件需要手动安装和更新，否则会报错无法启动 QQ，建议作为插件/主题查看器。
3. [轻量工具箱 —— 轻量 · 优雅 · 高效 · 福瑞](https://github.com/xiyuesaves/LiteLoaderQQNT-lite_tools/tree/v4)：聚合了大量功能的工具箱，免于四处寻找插件。部分可选功能如下：
   - **美化聊天界面**实现类 tg 效果（显示头像，加时间戳，消息靠左等）
   - 移除称号、VIP 、推荐标签等乱七八糟的东西。
   - 右键快捷搜索文字/图片，消息转图片发送
   - 选项高亮，特殊消息高亮
   - **小程序分享转 URL 卡片**，记录离开时位置，快捷+1（复读机）
   - **消息预览**：根据消息中第一个链接生成一张类 tg 的预览卡片
   - 本地表情
   - 消息后缀
   - **撤回消息缓存并高亮**
   - **设置背景**，调整亮度、透明度，可加磨砂等效果
   - **精简侧边栏**，所有功能可开关
   - 输入框、消息框功能开关
   - ……
4. [二维码解析](https://github.com/xh321/LiteLoaderQQNT-QR-Decode/tree/master)：对 QQNT 聊天中的图片进行二维码解析
5. [使用自定义浏览器打开链接并跳过拦截页](https://github.com/xh321/LiteLoaderQQNT-Directly-Jump/tree/master)
6. [将 DeepL 翻译接入你的 QQNT](https://github.com/MUKAPP/LiteLoaderQQNT-DeepL/tree/main)
7. [Markdown](https://github.com/d0j1a1701/LiteLoaderQQNT-Markdown/tree/v4)：为 QQ 添加 Markdown 渲染支持。发出的消息只有安装了该插件的 QQNT 能渲染 markdown，所以大部分情况下没用。
8. [Kill-Update](https://github.com/xh321/LiteLoaderQQNT-Kill-Update/tree/master)：关闭 QQ自动更新弹窗，有些插件不会第一时间支持最新版 QQ，所以禁止更新是有用的。
9. [window-on-top](https://github.com/mo-jinran/window-on-top/tree/v4)：让窗口有置顶功能

还有一些主题我没有介绍，因为去除垃圾之后部分美化的 QQNT 已经很好看了。
另外 ChatGPT 等 AI 接入插件也有，懒得折腾 API（~~没钱~~）。

## 网易云音乐美化
==特别推荐！==
众所周知，国内几大音乐平台巨头都在走复杂化、流量化路线，今天加一个社区，明天加一个短视频，长期 VIP 跳脸。
作为听音乐的软件，有几项基本功能就够了。国内外都有很多做得好的音乐播放器，但“播放器”和“音乐平台”之间还是有很多差距的。比如便捷搜索歌曲，查看评论，一起听等。建立歌单、关注歌手、充值购买等沉没成本也迫使用户继续吃屎。
我一直用的网易云音乐有一群伟大的互联网人做了插件，美化后确实美观好用。
首先是插件平台 **betterncm**
- 官网：[MicroBlock | BetterNCM](https://microblock.cc/betterncm) 
- GitHub 项目地址：[GitHub - MicroCBer/BetterNCM: NCM 软件插件管理器](https://github.com/MicroCBer/BetterNCM)
- 社区：[关于 BetterNCM 最全面的介绍以及疑难解答 - 飞书云文档](https://v2e2npdz15.feishu.cn/docx/UZkSd9d46o4fVOxaPNBcGXSenme)

优点是安装 betterncm 后，所有的主题、插件可以从网易云内平台下载、更新，不用一个个去查找翻 GitHub。
**推荐主题：**
- Materia You：比较简洁的主题，纯色背景没有背景图片。配色方案多样。
![Matreia You](https://github.com/user-attachments/assets/806686db-ace6-424b-b5af-1963fa18ffc4)

**推荐插件：**
1. RoundCornerNCM：网易云音乐窗口圆角，仅 Windows 11。
2. MikuPlugin：管理各元素是否显示，可以关闭恼人的视频、直播等元素。
![MikuPlugin](https://github.com/user-attachments/assets/fad6a44f-9ad0-4ffa-817e-8a4fe065de19)

3. 类苹果歌词：将歌曲页面改为类 Apple Music 的样式，还可更改歌词源。
![类苹果歌词](https://github.com/user-attachments/assets/1b868a40-004a-4e86-9ca5-7115a544fd36)

4. 这首歌的封面是什么？：歌曲列表添加封面，增加储存占用，易造成卡顿
![这首歌的封面是什么？](https://github.com/user-attachments/assets/aafa26cc-6c98-49d7-8ad7-28b59b1af954)

5. RuLyrics：桌面歌词插件，支持逐词，主副歌词，更改字体，分别更改前景色（已唱）、背景色（未唱），支持嵌入任务栏（与 TranslucentTB 同时使用似乎有点问题）
![桌面歌词](https://github.com/user-attachments/assets/3f2ab117-4c17-47b6-80c6-c1cdcbca92bd)

6. 更多好用插件自行探索下载
7. 部分插件对其他插件有依赖、冲突，注意在 GitHub Issues 查看甄别。

## Wallpaper Engine
==必备好物！==
鼎鼎大名的万能小红车，开启××√享受崭新人生（bushi）
作为最常用最好用的壁纸软件，wallpaper 有许多优点：
- 资源丰富。steam 创意工坊每时每刻都有大量优质壁纸上架，且几乎全部免费下载使用。
- 资源种类多。视频、图片、动图、网页……壁纸种类极多，且不少壁纸功能丰富，聚合了音乐歌词、特效、频谱等功能，一键装点桌面。
- 找资源方便。Wallpaper 搜索和筛选规则完善丰富，可筛选壁纸分辨率、类型、适用年龄等。
- 使用简便。基于 steam 创意工坊，能访问 steam 就能下载壁纸，无需翻墙，下载速度有保障。大部分壁纸，即使功能复杂，在 wallpaper 界面也能轻松设置。
- 串联手机。Wallpaper 推出了安卓应用，可通过 PC 向手机传输使用壁纸。

**Warning**：部分复杂特效网页、高清视频壁纸较为消耗 GPU 性能，占显存较多。可以在 wallpaper 设置里调整帧率、特效和应用行为等改善。

唯一的缺点是在 steam 购买 wallpaper 需要 19 RMB，不过这价格真不高吧。

Steam：[Steam 上的 Wallpaper Engine：壁纸引擎](https://store.steampowered.com/app/431960/Wallpaper_Engine/)

## TranslucentTB
==特别推荐！==
任务栏透明工具，可以全透、亚克力、不透，换主题色等。
占用内存、储存极小，几乎不消耗 CPU 性能。

GitHub 项目地址：[GitHub - TranslucentTB](https://github.com/TranslucentTB/TranslucentTB)
GitHub 中文翻译项目地址：[GitHub - kasuganosoras/TranslucentTB-CN](https://github.com/kasuganosoras/TranslucentTB-CN)

## Rainmeter
==特别推荐！==
久负盛名的桌面组件工具，可以自制功能多样的**小部件**放在桌面，也可以方便地导入他人制作的部件（即皮肤）。
常见的功能有：
- 显示 CPU、GPU、内存等硬件信息（实时更新）
- 音频识别生成各式各样的律动频谱条
- 媒体播放器
- 一键追番、查看新番
- 放置图片、轮播图库等
- ……

缺点
- 组件放多了会卡
- 部分组件资源消耗大
- 选择太多了需要折腾

官网：[Rainmeter](https://www.rainmeter.net/)
中国官网：[个人分享 Rainmeter](https://rainmeter.cn/)
GitHub 项目地址：[GitHub - rainmeter](https://github.com/rainmeter/rainmeter)
中国社区：[雨滴美化社区](https://bbs.rainmeter.cn/)

## Start 11
任务栏及开始菜单美化工具
- 可以将开始菜单改为 Windows 7-11 的风格，可改变颜色、透明度、间距、对齐方式等。
- **高级索引功能**：与 Edge 浏览器配对时，打开的选项卡也会显示在搜索内容中，最常用的内容在结果中的显示位置会更高。**移除本地搜索结果旁边显示的 Web 内容的选项！**
- 更改开始按钮图标，开始菜单背景图，任务栏颜色、纹理。

缺点
- 付费
- 我用起来有卡顿、启动慢等现象。
- 和 TranslucentTB 不兼容。

官网终身版售价 35 RMB，小贵，30 天免费试用。有许多代理，存在学习版（不推荐）。

官网：[Start11](https://stardock-start11.com/)
中国官网： [Start 11](https://www.start11.cn/)
Steam（褒贬不一）：[Steam 上的 Start11 v2](https://store.steampowered.com/app/1694750/Start11_v2/)

## 枫の美化工具箱
文件资源管理器、开始菜单、全局窗口美化工具
目前拥有的功能：  
- 主页面：自定义文件资源管理器窗口字体、局部颜色模式 Light、Dark (实验性局部Dark模式存在小的视觉bug)  
- 背景设置：自定义文件资源管理器、开始菜单、系统设置的**背景图片**  
- 颜色设置：自定义文件资源管理器配色(标题、组、页眉、详细信息、硬盘进度条)、圆角化硬盘进度条  
- 图标设置：自定义桌面、文件资源管理器图标组图标  
- 窗口设置：自定义文件资源管理器窗口**背景效果**半透明、Blur、Acrylic、Mica效果 win11圆角类型、开始菜单、系统设置Acrylic背景效果  
- 控件样式：自定义全局窗口**标题栏按钮样式**、macOS样式按钮、文件资源管理器Tab标签页、**工具栏**、地址栏配色、自绘圆角滚动条  
- 预设列表：保存、导入、分享你的配置文件  
- 插件列表：安装插件增强和扩展工具箱的功能

Evaluations：
- 还算是简单易用，内存和 GPU 占用尚可接受。
- 可装插件和导入配置文件，有一定可玩性。
- 不兼容 TranslucentTB
- 我用起来文件资源管理器常崩溃、不显示背景图片、工具栏配色未更改，可能已经优化稳定了。
- 背景图片清晰度不高，在 dark 模式下背景图片容易干扰正常阅读文件信息
- 软件免费，不完全开源，需要注册登录账号。

发布地址：[win美化工具箱 ★ 枫の主题社](https://winmoes.com/tools/12948.html)

## Sapphire
==特别推荐！==
桌面图标、布局美化，**更改交互体验**。
- 可调整桌面图标布局，更改横纵网格数
- 放置格子（类手机上的文件夹），将桌面各类文件直接放进文件夹，分类同一管理
- 调整图标、格子大小（网格数整数倍），圆角，背景，透明度
- 更改文件名的字体
- 精简模式去除文件名，可每个图标单独设置
- 可设置交互动效
- **Dock 栏**，类 Mac/手机下方的 dock，可竖置，改变长宽、背景色
- 开启文件夹高级交互后可以文件夹鼠标悬停预览内容，单击原地展开，不必开文件资源管理器窗口。
- **针对 steam 内应用的快捷方式优化**
- **双击隐藏图标**，每个可单独设置是否隐藏。
- 后台内存占用 100 mb 左右，尚可接受。几乎不消耗 GPU
- 可自定义图片、动图作为壁纸。初步兼容 WallpaperEngine（需在设置中设置当其他程序成为焦点时为始终运行）。
- 多屏支持

缺点：
- 我的电脑开启高级文件交互后，只要文件夹内文件数稍多，sapphire 会直接卡死。观望作者更新。
- 暂时还没有保存桌面布局的功能，好在即使卡退也会记录最新布局。
- 只保证兼容 Windows 11
- 更改桌面右键交互但目前还不能自定义

GitHub 项目地址：[Sapphire-EnhancedDesktop: Windows桌面取代软件](https://github.com/hymnly133/Sapphire-EnhancedDesktop)

## 光标美化
网上有很多分享光标美化文件的，根据个人喜好挑选。安装也不难。
分享两款，不影响视野同时具有个性化设计的：
1. 原神纳西妲光标：[纳西妲同款鼠标光标4.0版来咯\_原神](https://www.bilibili.com/video/BV14P411c7ap/)
2. 蔚蓝档案千年光标，简约可爱，在用。
   GitHub项目地址：[BlueArchive-Cursors](https://github.com/makipom/BlueArchive-Cursors)

**Comment by mztxy229** [comment]
看不懂，但是感觉很nb

**Comment by March-777th** [comment]
刚好想搞下美化<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >

**Comment by March-777th** [comment]
就是这个DeepL翻译支持DeepLX吗？<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >

**Reply by GuZhengSVT** [reply]
支持
![image](https://github.com/user-attachments/assets/c2ad6e20-fd35-42d1-bc83-653d80f6ae68)

**Comment by Carkys** [comment]
几个月前bing壁纸突然不工作了，整个电脑背景全黑，当时也懒得整，现在发现全黑的背景还挺舒服的，桌面上的图标都贼明显

**Comment by owofile** [comment]
很详细的分享！感谢！我都想关注你的博客了！ wwww

**Reply by GuZhengSVT** [reply]
https://guzhengsvt.top
嘿嘿

**Comment by AuroraBorealis567** [comment]
好棒
